templates/deputy/users_message.html.twig

```diff
@@ -0,0 +1,59 @@
+{% extends 'deputy/_layout.html.twig' %}
+
+{% block head_javascripts %}
+    <script type="text/javascript" src={{ asset('bundles/sonataadmin/vendor/jquery/dist/jquery.min.js') }}></script>
+{% endblock %}
+
+{% block javascripts %}
+    {% import 'javascript.js.twig' as js %}
+
+    <script src="{{ asset('ckeditor/ckeditor.js') }}"></script>
+    <script src="{{ asset('ckeditor/lang/fr.js') }}"></script>
+    <script type="text/javascript">
+        Kernel.onLoad(function() {
+            {{ js.ckeditor('deputy_message_content', null, false, 5000) }}
+        });
+    </script>
+{% endblock %}
+
+{% block deputy_content %}
+    <div class="l__wrapper--slim">
+        <h3>Envoyer un message</h3>
+
+        <div class="space--30-0">
+            {{ form_start(form) }}
+            {{ form_errors(form) }}
+
+            <div class="form__row">
+                {{ form_label(form.subject, 'Destinataires (n\'inclut que les abonnés aux e-mails)') }}
+                <input id="recipient" type="text" disabled="disabled" class="form form__field form--full" value="{{ results_count }} marcheurs(s)" />
```

Comment:
Why is is not part of the FormType ?

Reply:
Because it's not part of the object, it's just the visual rendering to make the representation homogeneous.